cddl/contrib/opensolaris/tools/ctf/cvt/merge.c

```diff
@@ -748,7 +748,7 @@ conjure_function(tdesc_t *old, int newselfid, merge_cb_data_t *mcd)
 	tdesc_t *new = conjure_template(old, newselfid);
 	fndef_t *nfn = xmalloc(sizeof (fndef_t));
 	fndef_t *ofn = old->t_fndef;
-	int i;
+	uint_t i;
```

Comment:
This is an unrelated change, and ought to go to https://github.com/openzfs/zfs

Reply:
Fixed!

Reply:
Though I don't think you posted exactly the right upstream...

Reply:
Ah, right you are.